extensions/sql-migration/src/constants/strings.ts

```diff
@@ -79,6 +79,7 @@ export const DATABASE_BACKUP_NC_NETWORK_SHARE_RADIO_LABEL = localize('sql.migrat
 export const DATABASE_BACKUP_NC_BLOB_STORAGE_RADIO_LABEL = localize('sql.migration.nc.blob.storage.radio.label', "My database backups are in an Azure Storage Blob Container");
 export const DATABASE_BACKUP_NETWORK_SHARE_HEADER_TEXT = localize('sql.migration.network.share.header.text', "Network share details");
 export const DATABASE_BACKUP_NC_NETWORK_SHARE_HELP_TEXT = localize('sql.migration.network.share.help.text', "Provide the network share location that contains backups and the user credentials that has read access to the share");
+export const DATABASE_BACKUP_NETWORK_SHARE_TABLE_HELP_TEXT = localize('sql.migration.network.share.storage.table.help', "Enter target database name for the selected source database(s).");
```

Comment:
should "database(s)." just be "database"? doesn't the target database name only apply to 'the selected source database."

Reply:
I think the UI mockups had "database(s)" because the user could be migrating more than one database. So the intention was to convey they need to enter the target name for each source db. I'll bring this up with Kathy to get her inputs on this. Thanks for bringing this up